medicines/doc-index-updater/src/create_manager/sftp_client.rs

```diff
@@ -20,7 +20,9 @@ pub enum SftpError {
 async fn sentinel_sftp_factory() -> Result<Sftp, SftpError> {
     let server = get_env_fail_fast("SENTINEL_SFTP_SERVER").await;
```

Comment:
how do I set `SENTINEL_SFTP_SERVER`?

Reply:
This is set in the `.env` file to localhost by default